spec/delete_host_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper' # spec_helper?
+require 'sidekiq/testing'
+
+describe DeleteHost do
+  let(:user) { FactoryBot.create(:user) }
+  let(:host) { FactoryBot.create(:host, org_id: user.account.org_id) }
+  let(:profile) { FactoryBot.create(:profile, :with_rules, account: user.account) }
+  let(:message) { {'id' => host.id, 'type' => 'delete'} }
+
+  before do
+    logger = double(Rails.logger)
+    allow(logger).to receive(:info)
+    allow(logger).to receive(:debug)
+    allow(Sidekiq).to receive(:logger)
+  end
+
+  DeleteHost.clear
+
+  context 'when the deletion succeeds' do
+    let(:test_result) { FactoryBot.create(:test_result, profile: profile, host: host) }
+    let(:rule_result) { FactoryBot.create(:rule_result, rule: profile.rules.sample, test_result: test_result, host: host) } 
+
+    it 'finds a host to delete' do
```

Comment:
This is great, but you might need a context when a host to be deleted doesn't exist.

Reply:
When an invalid host is to be deleted, an exception is **not** being generated. Is that the behavior we want?

Reply:
Yup, it should not fail with an error :)